tests/ui/test_pvc_ui.py

```diff
@@ -23,22 +23,37 @@ def teardown(self):
     @ui
     @skipif_ocs_version("<4.6")
     @pytest.mark.parametrize(
-        argnames=["sc_type", "pvc_name", "access_mode", "pvc_size"],
+        argnames=["sc_type", "pvc_name", "access_mode", "pvc_size", "vol_mode"],
         argvalues=[
             pytest.param(
-                *["ocs-storagecluster-cephfs", "test-pvc-fs", "ReadWriteMany", "2"]
+                *["ocs-storagecluster-cephfs", "test-pvc-fs", "ReadWriteMany", "2", None]
+            ),
+            pytest.param(
+                *["ocs-storagecluster-ceph-rbd", "test-pvc-rbd", "ReadWriteMany", "3", "Filesystem"]
+            ),
+            pytest.param(
+                *["ocs-storagecluster-ceph-rbd-thick", "test-pvc-rbd-thick", "ReadWriteMany", "4", "Filesystem"]
+            ),
+            pytest.param(
+                *["ocs-storagecluster-cephfs", "test-pvc-fs", "ReadWriteOnce", "10", None]
+            ),
+            pytest.param(
+                *["ocs-storagecluster-ceph-rbd", "test-pvc-rbd", "ReadWriteOnce", "11", "Block"]
```

Comment:
I think it would be ideal to have both Block and Filesystem rbd PVCs with ReadWriteOnce and check that the volume mode of the PVC that gets created really matches the chosen volume mode.

Reply:
added